### PR TITLE
Add first version of landing page clone

### PR DESCRIPTION
**Pull request by karlhorky**
@karlhorky ready for review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "upflow-landing-clone",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/core": "^10.0.28",
+    "@testing-library/jest-dom": "^4.2.4",
```
Comment (karlhorky) [inline package.json:7]: You are not using this dependency, please remove it

File: package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "upflow-landing-clone",
+  "version": "0.1.0",
```
Comment (karlhorky) [inline package.json]:
```suggestion
  "version": "1.0.0",
```

**Review by JonasArtmeier — APPROVED** [review]
LGTM

**Review by hamedJenabi — APPROVED** [review]
LGTM

**Pull request merged**